airflow/www/security.py

```diff
@@ -684,7 +684,7 @@ def sync_perm_for_dag(
             self.log.info("Syncing DAG-level permissions for DAG '%s'", dag_resource_name)
             self._sync_dag_view_permissions(dag_resource_name, access_control)
         else:
-            self.log.info(
+            self.log.debug(
```

Comment:
I think we should also change it to debug 3 lines above.

Reply:
you're totally right, done!